bin/reth/src/commands/import.rs

```diff
@@ -136,6 +142,15 @@ impl ImportCommand {
         // open file
         let mut reader = ChunkedFileReader::new(&self.path, self.chunk_len).await?;
 
+        let mut start_header: Option<SealedHeader> = None;
+
+        if self.start != 0 {
+            start_header = provider_factory
+                .provider()?
+                .sealed_header(block_num)
+                .expect("start block is not canonical with db");
+        }
```

Comment:
we need to query for last available header in the storage (see how headers stage does it), no need for cli arg